sormas-backend/src/main/java/de/symeda/sormas/backend/geo/GeoShapeProviderEjb.java

```diff
@@ -382,6 +383,14 @@ private void buildCountryShape() {
 			}
 		}
 
+		// fixme: this should be removed if the above logic is rewritten/fixed
```

Comment:
Something like: "fixme: Remove when the shapefiles are no longer part of the resources. In this case the logic to build country shapes will have to be checked again to fix the problem is workarround by this."